src/variables.jl

```diff
@@ -223,6 +223,52 @@ function set_name(v::VariableRef, s::String)
     return MOI.set(owner_model(v), MOI.VariableName(), v, s)
 end
 
+"""
+    variable_with_name(model::AbstractModel, name::String)::AbstractVariableRef
+
+Returns the reference of the variable with name `name`. Throws an error if
+several variables have the same name (even if it is different from `name`).
```

Comment:
> Throws an error if several variables have the same name

I thought we changed this behavior. It's only an error if multiple variables have the name that you're looking for, right?

Reply:
Indeed!